docs/api-guide/views.md

```diff
@@ -184,6 +184,28 @@ The available decorators are:
 
 Each of these decorators takes a single argument which must be a list or tuple of classes.
 
+
+## View schema decorator
+
+To override the default schema generation for function based views you may use
+the `@schema` decorator. This must come *after* (below) the `@api_view`
+decorator. For example:
+
+    from rest_framework.decorators import api_view, schema
+    from rest_framework.schemas import AutoSchema
+
+    class CustomAutoSchema(AutoSchema):
+        def get_link(*args):
```

Comment:
I *guess* we oughta include the arg spec fully here(?) (Rather than just *args)

Reply:
5b8dcad510147b4aa736c8e546a3a29c3d57a34a